base/main_test_bucket_pool.go

```diff
@@ -503,6 +539,8 @@ func (tbp *TestBucketPool) createTestBuckets(numBuckets, bucketQuotaMB int, buck
 
 			tbp.emptyPreparedStatements(ctx, bucket)
 
+			tbp.setXDCRBucketSetting(ctx, bucket)
```

Comment:
I think that I'd be inclined to put the logic here to test if we should set `enableCrossClusterVersioningEnabled`.

If this is rosmar, we are going to skip this value and I don't think this is worth logging.